proto/wechaty/puppet/event.proto

```diff
@@ -24,6 +24,12 @@ enum EventType {
   EVENT_TYPE_RESET       = 24;
   EVENT_TYPE_LOGIN       = 25;
   EVENT_TYPE_LOGOUT      = 26;
+
+  EVENT_TYPE_DIRTY_CONTACT = 27;
```

Comment:
Appreciate your quick update!

While I'm refactoring the Puppet class now, I believe that we can just add one event type `EVENT_TYPE_DIRTY` instead of adding 5 different events.

The reason that we can just keep one event here, is that we can pass the `EventType` in the payload.

Please update, thank you very much.

Reply:
Okay, that makes sense.

One more question: where do you want to define the `EventType` that indicate the different dirty event type? Shall we create another enum?

Reply:
I think so.

A `PayloadType` has been created in my PR to wechaty-puppet, please have a look over there.

Reply:
Seems like this will introduce a lose constraint to our grpc proto, since the type definition is in `wechaty-puppet`'s ts code, and we can only use a `number` here in the method definition, is this okay for you?

Reply:
NO, introducing a loose constraint to our grpc proto is not acceptable.

We should create a new enum PayloadType in our GRPC Proto so that we can map from the enum number in TS to the enum number to Proto.

We have done this kind of before, please refer to read the related proto definition like `ContactGender` and `ContactType`.

Reply:
Yeah, I agree.

I think I misunderstood your comments above, thought you were going to only use the `PayloadType` in `wechaty-puppet`.

Okay, I will add a `PayloadType` into the proto according to the design in `wechaty-puppet`.

Reply:
Awesome, I believe we are very clear with this design now.

I hope we can start merging our PRs after you finish adding the `PayloadType` to our proto.